lib/data_logger/logger.ex

```diff
@@ -102,7 +78,47 @@ defmodule DataLogger.Logger do
     %{state | tasks: Map.put_new(tasks, ref, pid)}
   end
 
+  defp log_data(
+         _,
+         prefix,
+         data,
+         %{module: destination, options: options} = state
+       ) do
+    destination
+    |> Kernel.apply(:send_data, [prefix, data, options])
```

Comment:
I would re-write this as `destination.send_data(prefix, data, options)`
I think it adds overhead and is recommended in the erlang documentation.

Read more [here](http://erlang.org/doc/man/erlang.html#apply-3).

Reply:
I agree now after writing similar code in the export work